lib/nimble_template/templates/template.ex

```diff
@@ -56,15 +58,28 @@ defmodule NimbleTemplate.Templates.Template do
     Mix.shell().cmd("npm install --prefix assets")
   end
 
-  defp suppress_necessary_credo_warnings(%Project{
-         base_module: base_module,
+  defp suppress_credo_warnings_for_base_project(%Project{base_module: base_module}) do
+    CredoHelper.disable_rule(
+      "lib/#{Macro.underscore(base_module)}.ex",
+      "CompassCredoPlugin.Check.DoSingleExpression"
+    )
+  end
+
+  defp suppress_credo_warnings_for_project_type(%Project{web_project?: true} = project) do
+    suppress_credo_warnings_for_phoenix_project(project)
+  end
```

Comment:
I actually intentionally left it like this for consistency with the two other surrounding functions with the same name because they had do...end blocks / multiple statements. It looked a bit out of place otherwise 😄 

But yeah I can change to single line 👍

Reply:
As I remember, I think you guys have added a credo rule for the 1-line function 🤔. Anyway, I think the change won't affect the consistency with other surrounding functions @liamstevens111

Reply:
These functions have been removed in 87d205bdc0d98879c094a7a7ca6b24c70509a5f3 and now just back to calling them explicitly